luigi/contrib/mrjob.py

```diff
@@ -0,0 +1,288 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2010 bit.ly, 2016 enreach.me
+#
+# Original author Daniel Frank, modified into luigi.contrib by Jyry Suvilehto
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+""" MRJob wrapper for Luigi
+
+This package provides a convenient Luigi wrapper for using `MRJob
+<http://packages.python.org/mrjob/>`_ steps in workflows. See `here
+<https://pythonhosted.org/mrjob/guides/runners.html#running-your-job-programmatically>`_
+for the feature used to implement this functionality.
+
+The wrapper doesn't attempt to be complete or to hide any complexity. The user
+is expected to know their way around MRJob and to configure it separately.
+
+.. code-block:: python
+
+    import luigi
+    from luigi.contrib.mrjob import MRJobTask
+    import mrjob
+
+    class MRJobWordCountTask(MRJobTask):
+        destination = luigi.Parameter()
+
+        job_cls = mrjob.examples.mr_word_freq_count.MRWordFreqCount
+
+        def mrjob_opts(self):
+            return {
+                '-r', 'inline'
+                }
+
+        def input(self):
```

Comment:
Ahh, OK I may have misunderstood something about the structure of requiring things. I'll have to look into it.